BAZEL.md

```diff
@@ -148,8 +148,8 @@ plugin referencing [ij.bazel.build][intellij_plugin].
 If the correct plugin does not exist in the list, then your IntelliJ version
 might be too recent, and the Bazel plugin might not have been upgraded to
 support it, yet. Check for version compatibility on the [Jetbrains plugin
-page][intellij_plugin_jetbrains]. At the time of writing the Bazel IntelliJ
-plugin version 2018.10.08.0.2 has been tested on IDEA Community 2018.2.5.
+page][intellij_plugin_jetbrains]. At the time of writing the Bazel plugin
+version 2020.04.13.0.0 has been tested on IDEA Community 2019.3.5.
 
 [intellij_plugin]: https://ij.bazel.build/
 [intellij_plugin_install]: https://ij.bazel.build/docs/bazel-plugin.html#getting-started
```

Comment:
worth adding a link to bazel plugin intellij compatibility page too so it's clear what versions of bazel plugin and intellij play nice?  especially the latest version of intellij basically never works well with bazel plugin since the plugin lags https://plugins.jetbrains.com/plugin/8609-bazel